Core/src/main/java/com/plotsquared/core/plot/flag/implementations/DisableBeaconEffectOverflowFlag.java

```diff
@@ -29,17 +29,17 @@
 import com.plotsquared.core.plot.flag.types.BooleanFlag;
 import org.checkerframework.checker.nullness.qual.NonNull;
 
-public class BeaconEffectFlag extends BooleanFlag<BeaconEffectFlag> {
+public class DisableBeaconEffectOverflowFlag extends BooleanFlag<DisableBeaconEffectOverflowFlag> {
```

Comment:
I thought it reasonable for the flag itself to prevent/enable all beacon effects on the plot, and only meant the config option should be renamed. Having the flag be true to allow/false to deny all beacon effects on the plot is more what players would want to do, whilst preventing overflow from/between plots is likely to be a more admin thing to do.

Reply:
Okay now I got what you meant. Changed it